Makefile

```diff
@@ -26,20 +26,40 @@ run: ## Run the wallabag built-in server
 build: ## Run webpack
 	@npm run build:$(ENV)
 
-test: ## Launch wallabag testsuite
-	@ant prepare && bin/simple-phpunit -v
+prepare: clean ## Prepare database for testsuite
+ifdef DB
+	cp app/config/tests/parameters_test.$(DB).yml app/config/parameters_test.yml
+endif
+	-php bin/console doctrine:database:drop --force --env=test
+	php bin/console doctrine:database:create --env=test
+ifndef DB ## make test does not define DB
+	php bin/console doctrine:schema:create --env=test
+endif
+ifeq ($(DB), sqlite)
+	php bin/console doctrine:schema:create --env=test
+endif
+ifeq ($(DB), mysql)
+	php bin/console doctrine:database:import data/sql/mysql_base.sql --env=test
+endif
+ifeq ($(DB), pgsql)
+	psql -h localhost -d wallabag_test -U travis -f data/sql/pgsql_base.sql
+endif
+
+fixtures: ## Load fixtures into database
+	php bin/console doctrine:fixtures:load --no-interaction --env=test
+
+test: prepare fixtures ## Launch wallabag testsuite
+	bin/simple-phpunit -v
 
 release: ## Create a package. Need a VERSION parameter (eg: `make release VERSION=master`).
 ifndef VERSION
 	$(error VERSION is not set)
 endif
 	@sh scripts/release.sh $(VERSION) $(TMP_FOLDER) $(RELEASE_FOLDER) $(ENV)
```

Comment:
Maybe, the bash scripts could also be integrated in makefile

Reply:
It's planned, I'll make a separate PR for that